HW1/P3/P3.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf
+
+conf = SparkConf().setAppName('P2').setMaster('local')
+sc = SparkContext(conf=conf)
+
+# Find combination of letters that can produce most anagrams
+
+def anagram(x):
```

Comment:
You should probably use different variable name (x as the parameter for anagram, and for the transformations may be confusing).